ansible/lifecycle_entry_point.yml

```diff
@@ -9,4 +9,12 @@
        })
     }}
 
+# Call livecycle_post_hook_ACTION
+- import_playbook: >-
+    {{ lookup('first_found', {
+         'files': [ 'lifecycle_hook_post_' + ACTION + '.yml', 'lifecycle_hook.yml' ],
```

Comment:
Shouldn't the generic hook in the config be named lifecycle_hook_post.yml  ?

`         'files': [ 'lifecycle_hook_post_' + ACTION + '.yml', 'lifecycle_hook_post.yml' ],`

Reply:
The idea was that this logic should work for all actions.
If naming livecycle_hook_post we'd have to have 3 defaults.
lifecycle_hook_post, lifecyclc_hook_pre, lifecycle_hook_status.
With just lifecycle_hook we have one default for the post hook regardless of action.